src/main/java/com/example/UserService.java

```diff
@@ -0,0 +1,29 @@
+package com.example;
+
+import java.util.List;
+
+public class userService {
+    private List<String> users;
+
+    // Constructor injection
+    public UserService(List<String> users) {
+        this.users = users;
+    }
+
+    // Rest of the code...
+
+    public boolean isUserExists(String userName) {
```

Comment:
nitpick: this is a bit grammatically weird
```suggestion
    public boolean doesUserExist(String userName) {
```